src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../../api/people';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState([] as Person[]);
+  const [loader, setLoader] = useState(false);
+
+  const peopleWithParents = (people: Person[]) => {
+    return people.map(person => ({
```

Comment:
avoid return

Reply:
but this function must return a new array with mapped data)